### DISCUSSION TITLE
"An error occurred reading this file." (Dreamcast OTA update)

### DISCUSSION THREAD

**Discussion by HornheaDD** [darthcloud/BlueRetro#965] [Q&A] [answered]
Hey there! I was wanting to update my BlueRetro adapter (made with the ESP32 board and installed internally, not purchased online)  and I keep getting the error in the title. 

I always have trouble remembering if I need the hw2 or hw1 because when I connect to the Dreamcast it says app_ver_is_hw2, but if I read correctly on the main BR page it says hw1 is the one I need. Regardless, when I select the file (either one) I get that error. Ive been trying to do it from my phone since my PC is too far away from the DC to pair. 

Im probably (as usual) missing something mundane, but any assistance would be great.  The screenshot cuts off part of the firmware name, but rest assured I tried with BlueRetro_hw2_dreamcast.bin and BlueRetro_hw1_dreamcast.bin and got the same result.

![Screenshot_20240418-140516](https://github.com/darthcloud/BlueRetro/assets/100097360/863abf4e-60e2-4893-9ddb-3547c8612a1a)

**Comment by darthcloud** [comment, marked as answer]
This is a Browser level error, somehow your web browser can't access the file.

**Comment by darthcloud** [comment]
If you don't know then it's HW1 ;)

**Comment by HornheaDD** [comment]
Ah ok, thanks man! I will try again but just move my DC closer to my PC :) Will report back when able!

**Comment by HornheaDD** [comment]
That was it! Not sure why my phone was being wonky, but my tablet (which I thought would do the same thing) worked just fine. Woo! Many thanks for your hard work DC!
